eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -187,24 +193,36 @@ spoAccepted re rs gas =
 -- vote are considered as `Abstain` votes.
 --
 -- `No` votes are not counted.
-spoAcceptedRatio :: forall era. RatifyEnv era -> GovActionState era -> Rational
+spoAcceptedRatio :: forall era. RatifyEnv era -> GovActionState era -> ProtVer -> Rational
 spoAcceptedRatio
-  RatifyEnv {reStakePoolDistr = PoolDistr individualPoolStake (CompactCoin totalActiveStake)}
+  RatifyEnv
+    { reStakePoolDistr = PoolDistr individualPoolStake (CompactCoin totalActiveStake)
+    , reDelegatees
+    , rePoolParams
+    }
   GovActionState
     { gasStakePoolVotes
     , gasProposalProcedure = ProposalProcedure {pProcGovAction}
     }
+  pv
     | totalActiveStake == 0 = 0 -- guard against the degenerate case when active stake is zero.
     | totalActiveStake == abstainStake = 0 -- guard against the degenerate case when all abstain.
     | otherwise = toInteger yesStake % toInteger (totalActiveStake - abstainStake)
     where
       accumStake (!yes, !abstain) poolId distr =
         let CompactCoin stake = individualTotalPoolStake distr
             vote = Map.lookup poolId gasStakePoolVotes
+            drep =
+              Map.lookup poolId rePoolParams >>= \d -> Map.lookup ((raCredential . ppRewardAccount) d) reDelegatees
          in case vote of
               Nothing
                 | HardForkInitiation {} <- pProcGovAction -> (yes, abstain)
-                | otherwise -> (yes, abstain + stake)
+                | bootstrapPhase pv -> (yes, abstain + stake)
+                | otherwise -> case drep of
```

Comment:
Would you mind either adding a comment here, explaining the idea - or maybe better yet, updating the haddock above to reflect this case.